pkg/sessions/redis/aws-iam/auth_test.go

```diff
@@ -0,0 +1,35 @@
+package auth
+
+import (
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestAWSIAMTokenGenerator(t *testing.T) {
```

Comment:
I saw another OSS project add a similar unit test when they added IAM auth for postgres:
https://github.com/authzed/spicedb/pull/1858/files